x-pack/test/functional/apps/machine_learning/create_single_metric_job.ts

```diff
@@ -138,9 +138,56 @@ export default function({ getService }: FtrProviderContext) {
     it('displays the created job in the job list', async () => {
       await ml.navigation.navigateToMl();
       await ml.navigation.navigateToJobManagement();
-      await ml.jobManagement.filterJobsTable(jobId);
-      const jobRow = await ml.jobManagement.getJobRowByJobId(jobId);
-      expect(jobRow).to.not.be(null);
+
+      await ml.jobTable.waitForJobsToLoad();
+      await ml.jobTable.filterWithSearchString(jobId);
+      const rows = await ml.jobTable.parseJobTable();
+      expect(rows.filter(row => row.id === jobId)).to.have.length(1);
+    });
+
+    it('displays details for the created job in the job list', async () => {
+      const rows = await ml.jobTable.parseJobTable();
+      const job = rows.filter(row => row.id === jobId)[0];
+      expect(job.description).to.eql(jobDescription);
+      expect(job.jobGroups).to.eql(jobGroups);
+      expect(job.recordCount).to.eql('2,399');
+      expect(job.memoryStatus).to.eql('ok');
+      expect(job.jobState).to.eql('closed');
+      expect(job.datafeedState).to.eql('stopped');
+      expect(job.latestTimestamp).to.eql('2016-02-11 23:56:59');
+
+      const countDetails = await ml.jobTable.parseJobCounts(jobId);
+      const counts = countDetails.counts;
+      expect(counts.job_id).to.eql(jobId);
```

Comment:
When an assertion like this fails you're going to see an error message like `expected 123 to equal 456`, if instead you structure your assertion like so you'll see more information that could help with debugging:
```js
expect(counts).to.eql({
  job_id: jobId,
  processed_record_count: '2,399'
  ...
})
```

Reply:
Good idea, thanks @spalger ! I've changed the validation statements.